crates/orchestrator/src/tests/jobs/state_update_job/mod.rs

```diff
@@ -51,30 +58,58 @@ async fn test_process_job() {
         let program_output: Vec<[u8; 32]> = vec![];
```

Comment:
I know it's not in this PR but instead of looping over the blocks, let's make these different test cases?

Reply:
Ig ese hi theek h vese bhi mock test h idts abhi ke lie necessary hoega. Vese bhi we are going to add more robust test cases ig uske baad update ho hi jaega as per the requirements.

Reply:
hmm, i think we should move it to different cases but we can do it another PR for sure. regarding the more robust cases, ya we definitely need to decide an exhaustive list of test cases needed but these tests would still remain